ci/jenkins/Nightly.groovy

```diff
@@ -203,6 +207,9 @@ pipeline {
                                                     mqMode='kafka'
                                                     tag="L0 L1 L2 ClusterOnly"
                                                     e2e_timeout_seconds = 6 * 60 * 60
+                                                } else if("${MILVUS_SERVER_TYPE}" == "authentication") {
+                                                    tag="L3"
```

Comment:
if this is used for all the "L3" cases now, you need to add all the extra configurations for all the "L3" cases.